doc/Extensions/Rancid.md

```diff
@@ -139,15 +139,14 @@ ls
 Make sure your config files are there :-)
 
 ```
-sudo usermod -a -G rancid www-data
-cd /opt/librenms/
+sudo usermod -a -G rancid librenms
 ```
 
 Add Rancid into LibreNMS config.php:
 
-```php
+```
 ### Rancid
-$config['rancid_configs'][]             = '/var/lib/rancid/librenms/configs/';
```

Comment:
Its suppose to be an array

Reply:
Ok. I've tested $config['rancid_configs']['']  which didn't work.

$config['rancid_configs'][]    does work, as well as $config['rancid_configs']